Essentials/src/config.yml

```diff
@@ -141,6 +141,9 @@ warn-on-smite: true
 # Shall we drop items instead of adding to inventory if the target inventory is full?
 drop-items-if-full: false
 
+# Shall we clear the offhand slot when executing the command /clear [player] [item]
+clear-offhand-slot: false
```

Comment:
It makes more sense to keep the old behaviour for old configs and change the behaviour for fresh configs, but this does the opposite?

Reply:
Wdym by that?

Reply:
He wanted that for new configs, it will be true (basically change this to true), however for existing ones it will be false (defined in `Settings.java`, change it to false there)

Reply:
I'm so dumb, I forgot to change the `isOffHandSlotClearingEnabled()` in `Settings.java` to false when I was thinking of changing `clear-offhand-slot` to false xD. Thanks!